### PR TITLE
feat: add URI variable, add encrypted token, add ./test.json to .…

### PR DESCRIPTION
**Pull request by umahaea**
feat: add URI variable, add encrypted token for triggering dependent build in .travis.yml file, add ./test.json to .gitignore

### CODE REVIEW

**Comment by umahaea** [comment]
@aaronkaka and @StommePoes - please review this PR - some code clean up in the shell_scripts.

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,4 @@ _gh_pages
 .DS_Store
 .idea
 test/fixtures/color/*.json
-.test.json
+./test.json
```
Comment (aaronkaka) [inline .gitignore]: just remove the ./ altogether, it's not needed

File: test/shell_scripts/run_tests.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ curl -s -X POST \
   -H "Content-Type: application/json" \
   -H "Accept: application/json" \
   -H "Travis-API-Version: 3" \
-  -H "Authorization: token Iit6_JxttCJ5JfEGQUyjRQ" \
+  -H "Authorization: token $AUTH_TOKEN" \
```
Comment (aaronkaka) [inline test/shell_scripts/run_tests.sh]: So does $AUTH_TOKEN automatically pick up the token from env in .travis.yml? If so, then WIN

File: .travis.yml [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
   skip_cleanup: true
   on:
     branch: v0
     tags: true
+env:
+  global:
+    secure: al6MU0Nb+7bsA6yeGoJb/gdvwxA0AlvpxJAawFfPUkYzkL3wS1jn6i9s/KEtRGRfKRkfR/Pl6YUMd45QXY3MbvTooAE9ZvmvZo24nqiYKSh3ZtLcu+uE7aOs9AgaRfpr4Uq1ezSr8wSt1An4aOUl+n9ULDkCJsMZEwcyUSZ9cGs19vSeKeSxi8gQBW2s0VMeeuqsZDMtsxzSa302EVaYzW0Ykpp7++zuSRNVrKPqToRv3eEsaKG7NxqLJqxIhNcXXC0+WzNal04zUoH0cfhoKghNZiBhzlKLtD+oGMpVyHt8sL2DZrB35ldw7RRz3jxQDUKxq78dyScAPX1HFuGXuSLCl4ttaIMR5jYcUQ6MUWOk+dOUYXG8OmhMUYI7Q9Mzt00O4Hf3b60GdzEW3IohVL4VA+rXu2npNc6oXVaO/h8ko/rxY1bpzFwWsaQgAanp4TU1C48KYTxyQM0r614F/l5kmIzZLFRVDo5vpuojOpz8k1v0zaQ5CNtHA7PAL7eONDlmi/wB5wbwTegsqOOEFd2p2rKLO8qOQDuSQ3R+GR9ssJsK5zTmTc3wfvhA8lKwQQOOGOAi87IDrmt+DZtJ9z/HirU6yA468RixrmWduw+rgOw5dI/uKqc2PuimM7HLGjPGCcA+aL+LSq1VIoiqA7h3W4LXIiQtDk1M3EX9Jww=
```
Comment (aaronkaka) [inline .travis.yml]: Was this a new token that you encrypted?

File: test/shell_scripts/run_tests.sh (hunk 2 above)
Comment (umahaea) [inline test/shell_scripts/run_tests.sh]: yes it doesn @aaronkaka .. I created a environment variable called AUTH_TOKEN
following this command:
>travis encrypt AUTH_TOKEN="my_git_hub_token" --add.
On its execution it automatically adds the secured encrypted token to .travis.yml file

File: .travis.yml (hunk 3 above)
Comment (umahaea) [inline .travis.yml]: yes this is the one @aaronkaka

File: test/shell_scripts/run_tests.sh (hunk 2 above)
Comment (umahaea) [inline test/shell_scripts/run_tests.sh]: and travis knows to pass this encrypted token to $AUTH_TOKEN during run time

**Pull request merged**